packages/server/src/sdk/app/backups/exports.ts

```diff
@@ -84,7 +84,11 @@ export async function exportDB(
 }
 
 function defineFilter(excludeRows?: boolean, excludeLogs?: boolean) {
-  const ids = [USER_METDATA_PREFIX, LINK_USER_METADATA_PREFIX]
+  const ids = [
+    USER_METDATA_PREFIX,
+    LINK_USER_METADATA_PREFIX,
+    AUTOMATION_LOG_PREFIX,
```

Comment:
It looks like it's possible for `ids` to contain `AUTOMATION_LOG_PREFIX` twice now if the conditional on line 95 evaluates to true. Is that intended?

Reply:
It is not!